src/pages/Home/Banner/index.tsx

```diff
@@ -15,9 +16,14 @@ const _Banner: FC<{ latestBlock?: State.Block }> = ({ latestBlock }) => {
     const container = ref.current
     if (!container) return
     try {
-      const r = createBannerRender(container)
-      setRender(r)
-      return () => r.destroy()
+      // The `tinyTestSceneRender` is used to determine whether the performance of browser is sufficient to render the three.js animated banner
+      // Related issue: https://github.com/Magickbase/ckb-explorer-public-issues/issues/218
+      const testRenderTime = tinyTestSceneRender(container)
+      if (testRenderTime < 60) {
+        const r = createBannerRender(container)
+        setRender(r)
+        return () => r.destroy()
+      }
```

Comment:
Oh, I forgot to mention that the "use new banner or not" option should be persisted here to avoid frequent execution of `tinyTestSceneRender`.

Reply:
> Oh, I forgot to mention that the "use new banner or not" option should be persisted here to avoid frequent execution of `tinyTestSceneRender`.

How about modify it to store the threshold in localstorage after the first visit